### ISSUE TITLE
No Alert shown to turn on bluetooth if bluetooth is off

### ISSUE THREAD

**Issue by Mouzmi** [Estimote/react-native-proximity#9]
When the bluetooth is Off the SDK is not showing any alert to navigate the user to turn bluetooth is on.   Is SDK supporting this or should we do separate code to check the bluetooth status.

**Comment by heypiotr** [comment]
You're right, for now, you need to do it yourself. Eventually, we should probably add it to the SDK (:
